Parse/Public/ParseClient.cs

```diff
@@ -122,7 +129,7 @@ public struct Configuration {
     /// </param>
     public static void Initialize(Configuration configuration) {
       lock (mutex) {
-        HostName = HostName ?? new Uri("https://api.parse.com/1/");
```

Comment:
You're right. I can update all of our internal tests to use Configuration.Server.